recipes/catch2/2.x.x/conanfile.py

```diff
@@ -44,6 +46,9 @@ def configure(self):
         if not self.options.with_main:
             del self.options.fPIC
             del self.options.with_benchmark
+        if self.options.default_reporter:
+            if '"' not in str(self.options.default_reporter):
+                self.options.default_reporter = '"{}"'.format(self.options.default_reporter)
```

Comment:
I would leave the options value alone,
and add a property method that returns a correctly parsed value:
e.g.
```python
    @property
    def _default_reporter_str(self):
        return self.options.default_reporter '"{}"'.format(self.options.default_reporter.strip('"'))
```

And use that `self._default_reporter_str` everywhere a string is required.

Reply:
Ok, fixed.